tasks/src/tabular-regression/about.md

```diff
@@ -12,6 +23,13 @@ Tabular regression is the task of predicting a numerical value given a set of at
 
 ## Useful Resources
 
+- Starting with Tabular Data Regression:
+    - Doing [EDA](https://neptune.ai/blog/exploratory-data-analysis-for-tabular-data) for Tabular data.
+    - Using [sklearn with Boston House data](https://amitg0161.medium.com/sklearn-linear-regression-tutorial-with-boston-house-dataset-cde74afd460a) to predict the value of a house.
```

Comment:
This dataset is known to contain racial biases and is removed officially from sklearn.

Reply:
My bad, I will rectify this.

Reply:
No worries! 🤗